main.js

```diff
@@ -0,0 +1,52 @@
+
+snakewaterMontana = {
+    guestOfHonor : "Dr. Alan Grant" ,
+    paleontologist: "Dr. Alan Grant",
+    depth: "10 meters",
+    specimen: "Velociraptor"
+  }
+    var cleverGirl = snakewaterMontana[specimen];
+    console.log(cleverGirl);
+
+    nicaragua = {
+      depth: "200 meters",
+      annualBudget: 1500000 + 250000,
+      specimens: [
+        "Tyrannosaurus Rex",
+        "Stegosaurous",
+        "Triceratops",
+        "Velociraptor"
+      ]
+  
+    }
+  nicaraguanSpecimens = specimens ;
```

Comment:
when you need to access the array inside an object to do that we use the `.` or the `[]` so the answer should be something like this `nicaragua.specimens` or what you did before  var nicaraguanSpecimens = `nicaragua["specimens"]